test/mocks/timeDate.js

```diff
@@ -1,8 +1,34 @@
-'use strict';
+function parseDate() {
+  return new Date();
+}
 
-module.exports = {
-  parseDate: () => new Date(),
-  timeDeltaToString: () => 'timeDelta',
-  fromNow: (d) => d.getTime() - Date.now(),
-  toNow: () => 35235,
+function timeDeltaToString() {
+  return 'timeDelta';
+}
+
+/* eslint no-use-before-define: "off" */
+/* eslint no-func-assign: "off" */
+// Disabled so we can manipulate the time string during tests.
+function setFromNowNeg() {
+  fromNow = () => -1;
+}
+
+function setFromNowPos() {
```

Comment:
i meant like a setter on the object

Reply:
Gotchya, I'll make some changes later tonight.